.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestHelpers.cs

```diff
@@ -200,9 +200,9 @@ public static IDateTimeExtractor GetEnglishExtractor(DateTimeExtractors extracto
                 case DateTimeExtractors.Set:
                     return new BaseSetExtractor(new EnglishSetExtractorConfiguration());
                 case DateTimeExtractors.Merged:
-                    return new BaseMergedExtractor(new EnglishMergedExtractorConfiguration(DateTimeOptions.None));
+                    return new DateTime.BaseMergedExtractor(new EnglishMergedExtractorConfiguration(DateTimeOptions.None));
```

Comment:
It's better to rename the DateTime version of BaseMergedExtractor to BaseMergedDateTimeExtractor